warehouse/oidc/models.py

```diff
@@ -180,8 +186,8 @@ def verify_claims(self, signed_claims: SignedClaims):
         # Finally, perform the actual claim verification.
         for claim_name, check in self.__verifiable_claims__.items():
             # All verifiable claims are mandatory. The absence of a missing
```

Comment:
The wording of this comment feels off, optional claims are also verifiable claims but are not mandatory right? I guess this is just using the name of the variable, but maybe that variable should be renamed too?

Reply:
Updated in 882c3e4!